shell/app/modules/cmp/common/alarm-strategy/index.tsx

```diff
@@ -646,7 +646,7 @@ export default ({ scopeType, scopeId }: IProps) => {
       render: (notifyGroup: COMMON_STRATEGY_NOTIFY.INotifyGroup) => {
         const tips = i18n.t('org:Notification group does not exist or has been remove. Please change one.');
         return (
-          <div className="flex-div">
+          <div className="flex-div flex">
```

Comment:
maybe we can remove the `flex-div` className and replace the style with tailwind
```
td.notify-info {
    .flex-div {
      justify-content: start;

      > span {
        width: 100%;
      }
    }
  }
```